### DISCUSSION TITLE
firebase 에서 오류

### DISCUSSION THREAD

**Discussion by ourgye** [velopert/dealing-with-react-native#35] [Q&A]
현재 Mac(intel코어) 사용 중이고, ios 빌드 중 문제가 발생합니다.
전 챕터까지는 정상적으로 작동하다가 firebase 챕터부터 오류가 나서 진도를 못나가고 있습니다.
cocoapods도 재설치하고, pod install --repo-update 도 하고, xcode 도 재설치 했는데... 무엇이 문제인지 모르겠습니다.

아래는 오류 메세지입니다 ㅜㅜ 

You have either:
 * changed the constraints of dependency `Firebase/Firestore` inside your development pod `RNFBFirestore`.
   You should run `pod update Firebase/Firestore` to apply changes you've made.
Couldn't install: Firebase/Firestore. Updating the Pods project and trying again…
Command `pod install` failed.
└─ Cause: CocoaPods could not find compatible versions for pod "Firebase/Firestore":
  In snapshot (Podfile.lock):
    Firebase/Firestore (= 8.5.0)

  In Podfile:
    RNFBFirestore (from `../node_modules/@react-native-firebase/firestore`) was resolved to 15.3.0, which depends on
      Firebase/Firestore (= 9.4.0)

You have either:
 * changed the constraints of dependency `Firebase/Firestore` inside your development pod `RNFBFirestore`.
   You should run `pod update Firebase/Firestore` to apply changes you've made.

**Comment by jihoonys82** [comment]
https://rnfirebase.io/ 여기서 getting start 페이지 보시면 설치방법이 나와 있습니다.
기존 방법에서 몇줄만 추가하시면 됩니다.
